apps/sms/views/inbox/js/inbox.js

```diff
@@ -983,6 +975,8 @@ var InboxView = {
   onDraftSaved: function inbox_onDraftSaved(draft) {
     var threadToUpdate = draft.threadId ? Threads.get(draft.threadId) : draft;
     this.updateThread(threadToUpdate);
+
+    this.showDraftSavedBanner();
```

Comment:
several issues here:
* in non-split views, this will run even if inbox is not shown, as you say.
* main issue I have is that we won't be able to show the info when we change the document...

I don't have a good solution yet.